### ISSUE TITLE
transcriber login sent to landing page

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#3597]
When we watched Dreanna log into FromThePage we were surprised that when she logged in -- as a transcriber -- she was taken to the landing page.  It should be the find a project page.

@anabeabas can you test this?  create a transcriber account, where are you taken?  log out, log back in, where are you taken?  (If it's a problem a quick video would be good to show it.)

Let's also test it when coming from an organization page (fromthepage.com/lva) or a specific work or collection.  You should be taken back to where you started the first time you login/create an account.  (If this is a problem, let's open a separate issue)

**Comment by saracarl** [comment]
(assign to @symmetrically if it's a real problem rather than something I'm remembering incorrectly.)

**Comment by anabeabas** [comment]
@saracarl @symmetrically 

I made a transcriber account and it took me to the landing page after I signed up. It did the same when I logged out and then logged back in. Video showing it here: https://www.loom.com/share/4f5b5c02b7114753be36ded57c9e8f60

I made another transcriber account from the LVA landing page (fromthepage.com/lva) and it took me back to the LVA landing page after I signed up, so no issues there.
